packages/terra-spacer/src/Spacer.scss

```diff
@@ -0,0 +1,335 @@
+$small-minus-2: 0.2142857143rem;
+$small-minus-1: 0.3571428571rem;
+$small: 0.5rem;
+$medium: 0.7142857143rem;
+$large: 0.8571428571rem;
+$large-plus-1: 1.0714285714rem;
+$large-plus-2: 1.4285714286rem;
+$large-plus-3: 2.1428571429rem;
+$large-plus-4: 3.5714285714rem;
+
+:local {
+  .margin-top-none {
+    margin-top: 0;
+  }
+
+  .margin-top-small-minus-2 {
+    margin-top: var(--terra-spacer-margin-top-small-minus-2, $small-minus-2); // 3px
```

Comment:
It seems like overkill to create theme-able variables for top, bottom, left, and right. Given the use of this component as a general spacer I think its highly unlikely that someone would want small-2 top to be different than small-2 bottom. I would think we'd create a theme-able variable for the small-2 etc variants and reuse it across top bottom left right. I could see different variables for margin vs padding, but I'm not totally convinced we need that level of complexity either.

Reply:
Can you provide a code sample of what you have in mind?

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/62058c71126f2b5fa04e334481e6e366d739ac4e#diff-bbd694561d6e058afb7ec7501feecb1aR16